guides/anchor/2-bridge-server.md

```diff
@@ -41,12 +41,12 @@ type = "mysql"  # or "postgres" if you created a postgres database
 url = "dbuser:dbpassword@/stellar_bridge"
 
 [accounts]
-# The secret seed for your base account, from which payments are made
+# The secret key for your base account, from which payments are made
 base_seed = "SAV75E2NK7Q5JZZLBBBNUPCIAKABN64HNHMDLD62SZWM6EBJ4R7CUNTZ"
```

Comment:
here's an example of a variable that needs to be changed if you want to standardize on `secret_key`.
`base_seed` -> `secret_key`